tests/analysis/adaptation/test_adaptation_option_collection.py

```diff
@@ -60,3 +60,20 @@ def test_calculate_options_unit_cost(
         # 3. Verify expectations.
         assert isinstance(_result, dict)
         assert all(_option in _result for _option in _collection.adaptation_options)
+
+    def test_calculate_correct_get_net_present_value_factor(
+        self,
+        valid_adaptation_config: tuple[AnalysisInputWrapper, AnalysisConfigWrapper],
+    ):
+        # 1. Define test data.
+        _collection = AdaptationOptionCollection.from_config(valid_adaptation_config[1])
```

Comment:
When only taking one argument of a tuple (instead of an object), it is best to check whether you have indeed the desired object. This way you prevent in the future from changing the output of `valid_adaptation_config` and then not being sure where the error is.


```suggestion
        # 1. Define test data.
        _input_wrapper = valid_adaptation_config[1]
        assert isinstance(_input_wrapper, AnalysisInputWrapper)
        _collection = AdaptationOptionCollection.from_config(_input_wrapper)
```